### PR TITLE
8327969: Convert javax/swing/border/Test6910490.java applet test to main

### PR DESCRIPTION
**Pull request by TejeshR13**
Convert javax/swing/border/Test6910490.java applet test to main based test using PassFailJFrame.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8327969](https://bugs.openjdk.org/browse/JDK-8327969): Convert javax/swing/border/Test6910490.java applet test to main (**Bug** - P4)

### Reviewers
 * [Damon Nguyen](https://openjdk.org/census#dnguyen) (@DamonGuy - Committer)
 * [Harshitha Onkar](https://openjdk.org/census#honkar) (@honkar-jdk - **Reviewer**) ⚠️ Review applies to [8e4729c3](https://git.openjdk.org/jdk/pull/18229/files/8e4729c39a1e4e3d06e907b935a8dc1c51c5e2bb)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/18229/head:pull/18229` \
`$ git checkout pull/18229`

Update a local copy of the PR: \
`$ git checkout pull/18229` \
`$ git pull https://git.openjdk.org/jdk.git pull/18229/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 18229`

View PR using the GUI difftool: \
`$ git pr show -t 18229`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/18229.diff">https://git.openjdk.org/jdk/pull/18229.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/18229#issuecomment-1991415206)

### CODE REVIEW

**Review by DamonGuy — APPROVED** [review]
Changes look straightforward and good to me.

File: test/jdk/javax/swing/border/Test6910490.java [hunk 1]
```diff
@@ -37,22 +37,40 @@
...
+                .build()
+                .awaitAndCheck();
+    }
 
-    @Override
-    public void init() {
+    public JFrame init() {
         Insets insets = new Insets(10, 10, 10, 10);
-        Dimension size = new Dimension(getWidth() / 2, getHeight());
+        JFrame frame = new JFrame("Matte border test");
```
Comment (DamonGuy) [inline test/jdk/javax/swing/border/Test6910490.java:63]:
```suggestion
        JFrame frame = new JFrame("Matte Border Test");
```

Comment (DamonGuy) [reply test/jdk/javax/swing/border/Test6910490.java:63]: Title case

**Review by DamonGuy — CHANGES_REQUESTED** [review]

**Review by honkar-jdk — APPROVED** [review]

**Review by DamonGuy — APPROVED** [review]

**Pull request closed without merging**